File: C#/02. Simple Operations and Calculations/Exercise/Alcohol Market.cs
using System;

namespace Alcohol_Market
{
    class Program
    {
        static void Main(string[] args)
        {
            double whiskeyPrice = double.Parse(Console.ReadLine());
            double beer = double.Parse(Console.ReadLine());
            double wine = double.Parse(Console.ReadLine());
            double rakiya = double.Parse(Console.ReadLine());
            double whiskey = double.Parse(Console.ReadLine());

            double rakiyaPrice = whiskeyPrice / 2;
            double winePrice = rakiyaPrice * (1 - 0.4);
            double beerPrice = rakiyaPrice * (1 - 0.8);

            double total = whiskeyPrice * whiskey + rakiyaPrice * rakiya + winePrice * wine + beerPrice * beer;
            Console.WriteLine($"{total:F2}");
        }
    }
}

File: Python/02. Simple Operations and Calculations/Exercise/Alcohol Market.py
whiskeyPrice = float(input())
beer = float(input())
wine = float(input())
rakiya = float(input())
whiskey = float(input())

rakiyaPrice = whiskeyPrice / 2
winePrice = rakiyaPrice * (1 - 0.4)
beerPrice = rakiyaPrice * (1 - 0.8)
total = whiskeyPrice * whiskey + rakiyaPrice * rakiya + winePrice * wine + beerPrice * beer

print(f"{total:.2f}")